src/MSALWrapper/AuthFlow/AuthFlowResult.cs

```diff
@@ -47,6 +47,11 @@ public AuthFlowResult(TokenResult tokenResult, IList<Exception> errors, string a
         /// </summary>
         public string AuthFlowName { get; internal set; }
 
+        /// <summary>
+        /// Gets or Sets the Duration in milli seconds.
+        /// </summary>
+        public double DurationInMs { get; set; }

```

Comment:
`ElapsedMilliseconds` returns a long. So we can use a long here too.
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.elapsedmilliseconds?view=net-6.0